maro/rl/training/trainer.py

```diff
@@ -23,7 +23,16 @@ class TrainerParams:
         automatically determined according to GPU availability.
     replay_memory_capacity (int, default=100000): Maximum capacity of the replay memory.
     batch_size (int, default=128): Training batch size.
-    data_parallelism (int, default=1): Degree of data parallelism.
+    data_parallelism (int, default=1): Degree of data parallelism. A value of greater than 1 can be used when
```

Comment:
`A value of greater than 1` -> `A value greater than 1` ?

Reply:
Yes, typo